static/.eslintrc.js

```diff
@@ -14,37 +11,37 @@ module.exports = {
         'promise/param-names': 'error',
         'promise/valid-params': 'error',
     },
-    overrides: [{
-        files: ['*.ts'],
-        plugins: [
-            'import',
-            '@typescript-eslint',
-        ],
-        extends: [
-            './.eslint-ce-static.yml',
-            'plugin:@typescript-eslint/eslint-recommended',
-            'plugin:@typescript-eslint/recommended',
-            'plugin:import/typescript',
-        ],
-        env: {
-            browser: true,
-            es6: true,
-            node: false,
-        },
-        parser: '@typescript-eslint/parser',
-        parserOptions: {
-            sourceType: 'module',
-            ecmaVersion: 'latest',
-            tsconfigRootDir: __dirname,
-            project: './tsconfig.json',
+    overrides: [
+        {
+            files: ['*.ts'],
+            plugins: ['import', '@typescript-eslint'],
+            extends: [
+                './.eslint-ce-static.yml',
+                'plugin:@typescript-eslint/eslint-recommended',
+                'plugin:@typescript-eslint/recommended',
+                'plugin:import/recommended',
```

Comment:
But added this line because it seems like we missed it. Nothing changed though so it might be wrong